Userland/Libraries/LibPDF/Renderer.h

```diff
@@ -263,6 +257,11 @@ class Renderer {
 
     RefPtr<Document> m_document;
     RefPtr<Gfx::Bitmap> m_bitmap;
+
+    RefPtr<Gfx::Bitmap> m_clipped_bitmap;
+    OwnPtr<Gfx::Painter> m_clipped_bitmap_painter;
+    OwnPtr<Gfx::AntiAliasingPainter> m_clipped_bitmap_anti_aliasing_painter;
```

Comment:
nit: Probably not worth making an `OwnPtr` for the `Gfx::AntiAliasingPainter` (`Optional` would probably do). `Gfx::AntiAliasingPainter` should only contain a `Painter&` + a transform.

Reply:
Made both the painter and the aa painter Optional<>s.